### PR TITLE
Updates iota token page re supply incr.

### PR DESCRIPTION
**Pull request by luca-moser**
Changes the supply numbers, links to the blog post, puts in warnings re the denomination on exchanges being used, adds a table for the different allocations and unlock schedules.

### CODE REVIEW

File: docs/get-started/introduction/iota/iota-token.md [hunk 1]
```diff
@@ -16,21 +16,54 @@ keywords:
 
 The IOTA token is the medium to transfer value within the IOTA mainnet. However, unlike other cryptos
 there are no fees to be paid with the base token, meaning that IOTA tokens are transferred 1 to 1 to the receiver,
-without miners or other intermediaries taking a cut.
+without miners or other intermediaries taking a cut. The maximum supply is 4,600,000,000 IOTAs.
 
-## Fair Distribution & No Inflation
+Note that the smallest denomination of IOTA is called "micros" with 6 decimal points. The total supply consists therefore
+out of 4,600,000,000 IOTA tokens or 4,600,000,000,000,000 micros.
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:22]:
```suggestion
The smallest denomination of IOTA is called "micros", with 6 decimal points. The total supply consists therefore
out of 4,600,000,000 IOTA tokens or 4,600,000,000,000,000 micros.
```

File: docs/get-started/introduction/iota/iota-token.md [hunk 2]
```diff
@@ -16,21 +16,54 @@ keywords:
...
 
-Unlike most other cryptocurrencies, all 2,779,530,283,277,76 IOTA tokens have been fairly distributed upon its
-initial ICO without any early VC funding or tokens allocated to founders. Furthermore, there are no vesting schedules
-or token lockups, meaning the entire supply is circulating.
+:::info
+Note that previously one micro was called one IOTA and 1,000,000 IOTAs made up 1 MIOTA (mega IOTA), however
+with the change to the supply and upgrade to the Stardust protocol on the 4th of October 2023 on the IOTA Mainnet, the
+IOTA Foundation opted to remove the [SI prefixes](https://en.wikipedia.org/wiki/Metric_prefix) in order to align with
+the more common denominations used in the DLT space.
+:::
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:29]:
```suggestion
:::info About MIOTA

Previously, one micro was called one IOTA, and 1,000,000 IOTAs made up 1 MIOTA (mega IOTA), however
with the change to the supply and upgrade to the Stardust protocol on the 4th of October 2023 on the IOTA Mainnet, the
IOTA Foundation opted to remove the [SI prefixes](https://en.wikipedia.org/wiki/Metric_prefix) to align with
the more common denominations used in the DLT space.

:::
```

File: docs/get-started/introduction/iota/iota-token.md [hunk 3]
```diff
@@ -16,21 +16,54 @@ keywords:
...
+
+## Supply Change With Stardust Upgrade
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:31]:
```suggestion
## Supply Change With the Stardust Upgrade
```

File: docs/get-started/introduction/iota/iota-token.md [hunk 4]
```diff
@@ -16,21 +16,54 @@ keywords:
...
+
+On the 4th of October 2023, the IOTA Mainnet was upgraded to run under the Stardust protocol. With the upgrade also a
+change in the supply was executed where the initial supply of 2,779,530,283,277,761 micros (~2,779,530,283 IOTAs) was
+increased to 4,600,000,000,000,000 micros (4,600,000,000 IOTAs) where the additional new supply of
+1,820,469,716,722,240 micros was distributed under following allocations and unlock schedules:
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:36]:
```suggestion
On the 4th of October 2023, the IOTA Mainnet was upgraded to run under the Stardust protocol. With the upgrade, a
change in the supply  was also executed, where the initial supply of 2,779,530,283,277,761 micros (~2,779,530,283 IOTAs) was
increased to 4,600,000,000,000,000 micros (4,600,000,000 IOTAs), where the additional new supply of
1,820,469,716,722,240 micros were distributed under the following allocations and unlock schedules:
```
Using micros may make the numbers a bit daunting. As an alternative:
```suggestion
On the 4th of October 2023, the IOTA Mainnet was upgraded to run under the Stardust protocol. With the upgrade, a
change in the supply  was also executed, where the initial supply of ~2,779,530,283 IOTAs (2,779,530,283,277,761 micros, to be precise) was
increased to 4,600,000,000 IOTAs (4,600,000,000,000,000 micros), where the additional new supply of
~1,820,469,713 IOTAs were distributed under the following allocations and unlock schedules:
```

File: docs/get-started/introduction/iota/iota-token.md [hunk 5]
```diff
@@ -16,21 +16,54 @@ keywords:
...
 
 ## Buying IOTA
 
-:::info
-Note that exchanges usually use the symbol MIOTA on their platforms which represents 1,000,000 (1 million) IOTA tokens.
+:::caution
+Note that some exchanges might still be using the symbol `MIOTA` on their platforms which represents 1,000,000 (1
+million) micro tokens. Before buying or selling IOTAs, make sure you understand what denomination the exchange is
+working with, so you don't get tricked into selling/buying under/over the current market price.
 :::
```
Comment (lucas-tortora) [inline docs/get-started/introduction/iota/iota-token.md:67]:
```suggestion
:::caution 

Note that some exchanges might still be using the symbol `MIOTA` on their platforms, which represents 1 IOTA or 1,000,000 (1
million) micros. Before you buy or sell IOTAs, please make sure you understand what denomination the exchange is
using so you don't get tricked into selling/buying under/over the current market price.

:::
```

**Review by lucas-tortora — APPROVED** [review]

File: docs/get-started/introduction/shimmer/shimmer-token.md [hunk 6]
```diff
@@ -26,11 +26,11 @@ out of 1,813,620,509 SMR tokens or 1,813,620,509,061,365 glow.
...
 The initial supply of SMR was distributed in the following way:
 
-| Allocation       | Percentage | Number of SMR Tokens | Purpose                                                                                                                           |
-| ---------------- | ---------- | -------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
-| IOTA Stakers     | 80%        | 1,450,896,407        | Rewarded to IOTA token holders who staked their tokens on the IOTA mainnet.                                                       |
-| Community DAO    | 10%        | 181,362,051          | To foster community involvement and governance, enabling members to contribute to the Shimmer ecosystem's development and growth. |
-| Tangle Ecosystem | 10%        | 181,362,051          | To support and incentivize projects that leverage the IOTA Tangle, contributing to the ecosystem's development.                   |
+| Allocation       | Percentage | # SMR Tokens  | Purpose                                                                                                                           |
```
Comment (Dr-Electron) [inline docs/get-started/introduction/shimmer/shimmer-token.md:29]: Shouldn't this be:
```suggestion
| Allocation       | Percentage | # of SMR Tokens  | Purpose                                                                                                                           |
```

**Review by Dr-Electron — DISMISSED** [review]

**Review by lucas-tortora — DISMISSED** [review]

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**